mods/ra/chrome/editor.yaml

```diff
@@ -172,6 +173,16 @@ Background@SAVE_MAP_PANEL:
 			Font: Bold
 			Key: escape
 
+ScrollPanel@MAP_SAVE_VISIBILITY_PANEL:
+	Width: 220
+	Height: 44
```

Comment:
Changed it back to 64. There's empty space now in the dropdown, but we can't have it both ways.